client/src/pages/Notification.js

```diff
@@ -22,10 +30,10 @@ class Notification extends Component {
         {notifications &&
           notifications.map((notif, i) => {
             return (
-              <div key={i}>
+              <div key={i} onClick={() => this.handleClicked(notif)}>
                 <div>message: {notif.message}</div>
                 <div>id: {notif._id}</div>
```

Comment:
For this part, I would only keep the message, createdAt, and the link property. The id doesn't mean anything to the person. The priority doesn't matter when read out like this (it preferably changes ordering or maybe the look of the actual content e.g. bold or some different colour outline). The read part should only be the orange dot and not typed out like this.